plugins/languages/texpresso.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+  helpers.vim-plugin.mkVimPlugin config {
```

Comment:
```suggestion
  # This plugin has no configuration, so we use `mkVimPlugin` without the
  # `globalPrefix` argument to avoid the creation of the `settings` option.
  helpers.vim-plugin.mkVimPlugin config {
```

Reply:
Done